crates/router/src/core/payments/flows/session_flow.rs

```diff
@@ -92,7 +92,7 @@ fn create_gpay_session_token(
 
     let session_data = router_data.request.clone();
     let transaction_info = payment_types::GpayTransactionInfo {
-        country_code: session_data.country.unwrap_or_else(|| "US".to_string()),
+        country_code: session_data.country.unwrap_or_default(),
```

Comment:
why did we remove "US" from default country?

Reply:
`US` is still the default country for `Country` enum. So unwrap or default returns US if option is None